src/compiler.js

```diff
@@ -1103,7 +1103,7 @@ var Compiler = Object.extend({
 // console.log(tmpl);
 
 module.exports = {
-    compile: function(src, asyncFilters, extensions, name, lexerTags) {
+    compile: function(src, asyncFilters, extensions, name, lexerTags, trimBlocks, lstripBlocks) {
```

Comment:
Problem is that path is not from opts and it's fourth argument. So either asyncFilters and extensions should be left where they are now. Or we break backwards compatibility.